examples/two-columns-layout/style.css

```diff
@@ -60,9 +60,162 @@ body {
   transition: background 0.2s ease-out;
 }
 
+.aa-Item:hover .aa-ItemPicture img {
+  transform: scale(1.1);
+}
+
+.aa-ItemPicture {
+  width: 100%;
+  overflow: hidden;
+}
+
+.aa-ItemPicture img {
+  object-fit: cover;
+  width: 100%;
+  height: auto;
+  border-radius: 3px;
+  transition: transform 1.8s ease-out;
+  transform-origin: center bottom;
+}
+
+/* Products */
+.aa-PanelSection--products {
+  padding: var(--aa-spacing-half);
+  transition: background 0.2s ease-out;
+}
+
+.aa-PanelSection--products-preview {
+  background: rgba(
+    var(--aa-selected-color-rgb),
+    var(--aa-selected-color-alpha)
+  );
+  border-radius: 3px;
+}
+
+.aa-PanelSection--products .aa-List {
+  display: flex;
+  column-gap: var(--aa-spacing);
+}
+
+.aa-PanelSection--products .aa-Item {
+  align-items: flex-start;
+  width: 100%;
+}
+
+.aa-PanelSection--products .aa-Item:hover .aa-ItemFavorite {
+  opacity: 1;
+}
+
+.aa-ProductItem {
+  height: 100%;
+  min-height: 380px;
+  align-items: flex-start;
+}
+
+.aa-ProductItem .aa-ItemPicture {
+  position: relative;
+}
+
+.aa-ProductItem .aa-ItemContentBody {
+  width: 100%;
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.aa-ProductItem .aa-ItemPictureBlurred {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  transition: opacity 0.2s ease-out, visibility 0s linear 0.2s;
+}
+
+.aa-ProductItem .aa-ItemPictureBlurred--loaded {
+  opacity: 0;
+  visibility: hidden;
+}
```

Comment:
Shouldn't we have `.aa-ItemPicture--blurred and .aa-ItemPicture--loaded` instead?